qiita_ware/commands.py

```diff
@@ -70,22 +71,41 @@ def submit_EBI(preprocessed_data_id, action, send, fastq_dir_fp=None):
     ebi_submission.write_xml_file(ebi_submission.generate_submission_xml(),
                                   ebi_submission.submission_xml_fp)
 
-    # other steps
     if send:
-        ebi_submission.send_sequences()
-        study_accession, submission_accession = ebi_submission.send_xml()
+        # step 4: sending sequences
+
+        # Set the ASCP password to the one in the Qiita config, but remember
+        # the old pass so that we can politely reset it
+        old_ascp_pass = environ.get('ASPERA_SCP_PASS', '')
```

Comment:
Ok, I guess we will need to be extremely careful once this branch is ready to be merged against master, since there is a lot of stuff that gets changed later and is getting harder and harder to review.

IMO we should remove those lines from here, as they will never be in here.